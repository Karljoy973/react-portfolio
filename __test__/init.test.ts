import { expect,  test } from 'vitest'

test("hello world", () => {
    const hello = "hello world"
    expect(hello).toBe("hello world")

})
test('tostring', () => {
const result = toString(0)
expect(result).toMatchSnapshot()
})


function toString(value: any) {
    return `${value}`
}