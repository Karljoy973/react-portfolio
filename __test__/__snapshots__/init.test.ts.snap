// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`tostring 1`] = `"0"`;
